docs/full-node-api/overview.md

```diff
@@ -2,37 +2,86 @@
+### What is a Full Node?
 
-We will shortly be upgrading the full node, to enable broadcasting transactions. 
+A full node plays a pivotal role in blockchain networks by validating transactions and blocks in accordance with the network's consensus rules. It contributes to the decentralization and security of the network, offering users a private and trustless interface to the blockchain.
 
-## Build and Run a Full Node
+## Bitfinity Full Node Overview
 
-To sync with the node you will need to build the code by following the instructions in https://github.com/bitfinity-network/erigon/blob/evmc_importer/BITFINITY.README.md on the evmc_importer branch. 
+The Ethereum Virtual Machine (EVM) supports a finite number of blocks. To access comprehensive historical data, you can operate a local full node, known as the block-importer.
 
-The block-importer uses two processes, one to download and validate blocks and an other to establish an RPC URL for the full node. For example: 
+### Bitfinity's Implementation
 
-``` 
-build/bin/blockimporter --evm <JSON_RPC_URL> --db <DB_PATH> &\
-build/bin/rpcdaemon --datadir <DB_PATH> --http.corsdomain * --http.api=eth,erigon,ots 
+Bitfinity offers a read-only full-node implementation that allows for accessing data on the EVM. Our full node, a derivative of Erigon, enables users to download historical blocks, process all transactions, and verify state roots. Additionally, the EVM can sign blocks, and the block-importer verifies these signatures to confirm the blocks' authenticity. We are in the process of enhancing the full node to include transaction broadcasting capabilities.
+
+## Getting Started with Your Full Node
+
+### Prerequisites
+
+Before initiating the setup, ensure you have:
+
+- A stable internet connection.
+- Adequate storage space for blockchain data.
+- Basic knowledge of terminal commands.
+
+## Setting Up and Running Your Full Node
+
+### 1. Clone the Repository
+
+Begin by cloning the Bitfinity full node repository from GitHub. Use the `evmc_importer` branch to access the latest features.
+
+```sh
+git clone -b evmc_importer https://github.com/bitfinity-network/erigon.git
 ```
 
-The JSON_RPC_URL must have all historical blocks.  You can retrieve all historical blocks from the following URLs: 
+For detailed build instructions, refer to the [Bitfinity Readme File](https://github.com/bitfinity-network/erigon/blob/evmc_importer/BITFINITY.README.md).
+
+### 2. Run the Full Node
+
+The block-importer operates through two processes: one for downloading and validating blocks, and another to set up an RPC URL for the full node.
```

Comment:
I'd rather put all this information in `https://github.com/bitfinity-network/erigon/blob/evmc_importer/BITFINITY.README.md` and just reference here